deploy/operator/kustomization.yaml

```diff
@@ -38,6 +38,14 @@ patches:
     group: apps
     kind: Deployment
     version: v1
+- patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/args/-
+      value: --images=ui-dashboards=quay.io/openshift-observability-ui/console-dashboards-plugin:v0.1.0
```

Comment:
not that it's any harm but this just passes the default value right doesn't it?

Reply:
yes, I wanted to reflect the new image for the plugin in the generated files in case some replacement needs it for the bundle productization

Reply:
Got it. For the productuzed CSV we will need to change this, but this can only happen in CPaaS, since we use the digest pinned image reference. Feel free to remove this here.